terraform/README-terraform.md

```diff
@@ -0,0 +1,245 @@
+---
+page_type: sample
+languages:
+- java
+products:
+- Azure Spring Cloud
+description: "Deploy Spring microservices using Azure Spring Cloud and MySQL"
+urlFragment: "spring-petclinic-microservices"
+---
+# Deploy Spring Microservices using Azure Spring Cloud and MySQL 
+
+Azure Spring Cloud enables you to easily run a Spring Boot based microservices application on Azure.
+
+This quickstart shows you how to deploy an existing Java Spring Cloud application to Azure. When you're finished, you can continue to manage the application via the Azure CLI or switch to using the Azure portal.
+
+## What will you experience
+You will:
+- Build existing Spring microservices applications
+- Provision an Azure Spring Cloud service instance
+- Deploy applications to Azure
+- Bind applications to Azure Database for MySQL
+- Open the application
+
+## What you will need
+
+In order to deploy a Java app to cloud, you need 
+an Azure subscription. If you do not already have an Azure 
+subscription, you can activate your 
+[MSDN subscriber benefits](https://azure.microsoft.com/pricing/member-offers/msdn-benefits-details/) 
+or sign up for a 
+[free Azure account]((https://azure.microsoft.com/free/)).
+
+In addition, you will need the following:
```

Comment:
Should we call out Terraform as pre-requisite?

Reply:
Added the Terraform as prerequisite for this readme.